xdsl/interactive/passes.py

```diff
@@ -37,14 +37,10 @@ def get_new_registered_context(
 
 
 def iter_condensed_passes(
+    ctx: Context,
     input: builtin.ModuleOp,
     all_passes: tuple[tuple[str, type[ModulePass]], ...],
 ):
-    ctx = Context(True)
-
-    for dialect_name, dialect_factory in get_all_dialects().items():
```

Comment:
Wow, I thought I got rid of all uses of `get_all_dialects()` when I was doing stuff downstream. I obviously missed one.